examples/tasks/mem-file.json

```diff
@@ -12,7 +12,11 @@
                 "/intel/procfs/meminfo/mem_total": {},
                 "/intel/procfs/meminfo/mem_used": {}
             },
-            "config": {},
+            "config": {
+	      "/intel/procfs": {
+                "proc_path": "/proc"
```

Comment:
I think it is best to keep example in a working state. This way, people can use the task right out of the box without any modifications. So /proc is the default which should be working and people can use any other customized value if need for use in docker for instance